src/main/java/com/instrumentalapp/Connection.java

```diff
@@ -273,7 +273,7 @@ private void backoffReconnect() throws InterruptedException {
 		// Do a reconnect, then try to send the message again.
 		// Since we've removed the message from the queue, don't discard it, and don't advance.
 		long delay = (long) Math.min(maxReconnectDelay, Math.pow(errors++, reconnectBackoff) * 1000);
-		LOG.severe("Connection failed to " + agentOptions.getHost() + ":" + agentOptions.getPort() + ". Retry in " + delay + "ms");
+		LOG.warning("Connection failed to " + agentOptions.getHost() + ":" + agentOptions.getPort() + ". Retry in " + delay + "ms");
```

Comment:
Generally speaking we should be using string formatting instaed of string concatenation, but that's like a problem for future us.

Reply:
Java!

Reply:
☕️